integration/benchnet2/automate/templates/helm-values-all-nodes.yml

```diff
@@ -0,0 +1,385 @@
+      args:
+        - --bootstrapdir=/bootstrap
+        - --datadir=/data/protocol
+        - --secretsdir=/data/secret
+        - --loglevel=INFO
+        - --profiler-enabled=false
+        - --profile-uploader-enabled=false
+        - --tracer-enabled=false
+        - --profiler-dir=/profiler
+        - --profiler-interval=2m
+        - --bind=0.0.0.0:3569
+        - --nodeid={{$val.node_id}}
+        - --rpc-addr=0.0.0.0:9000
+        - --secure-rpc-addr=0.0.0.0:9001
+        - --http-addr=0.0.0.0:8000
+        - --collection-ingress-port=9000
+        - --supports-observer=true
+        - --public-network-address=0.0.0.0:1234
+        - --log-tx-time-to-finalized
+        - --log-tx-time-to-executed
+        - --log-tx-time-to-finalized-executed
+      env:
+        - name: OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
+          value: http://tempo:4317
+        - name: OTEL_EXPORTER_OTLP_TRACES_INSECURE
+          value: "true"
+        - name: OTEL_RESOURCE_ATTRIBUTES
+          value: network
+        - name: BINSTAT_ENABLE
+          value: ""
+        - name: BINSTAT_LEN_WHAT
+          value: ""
+        - name: BINSTAT_DMP_NAME
+          value: ""
+        - name: BINSTAT_DMP_PATH
+          value: ""
+      image: gcr.io/flow-container-registry/access:v0.27.6
+      nodeId: {{$val.node_id}}
+{{end}}{{end}}
+collection:
+  defaults:
+    imagePullPolicy: Always
+    containerPorts:
+      - name: ptp
+        containerPort: 3569
+      - name: grpc
+        containerPort: 9000
+      - name: secure-grpc
+        containerPort: 9001
+      - name: admin
+        containerPort: 9002
+    env: []
+    resources:
+      requests:
+        cpu: "200m"
+        memory: "128Mi"
+      limits:
+        cpu: "800m"
+        memory: "10Gi"
+    servicePorts:
+      - name: ptp
+        protocol: TCP
+        port: 3569
+        targetPort: ptp
+      - name: grpc
+        protocol: TCP
+        port: 9000
+        targetPort: grpc
+      - name: secure-grpc
+        protocol: TCP
+        port: 9001
+        targetPort: secure-grpc
+      - name: admin
+        protocol: TCP
+        port: 9002
+        targetPort: admin
+    storage: 1G
+  nodes:
+{{range $val := .}}{{if eq ($val.role) ("collection")}}
+    {{$val.name}}:
+      args:
+        - --bootstrapdir=/bootstrap
+        - --datadir=/data/protocol
+        - --secretsdir=/data/secret
+        - --loglevel=INFO
+        - --profiler-enabled=false
+        - --profile-uploader-enabled=false
+        - --tracer-enabled=false
+        - --bind=0.0.0.0:3569
+        - --profiler-dir=/profiler
+        - --profiler-interval=2m
+        - --nodeid={{$val.node_id}}
+        - --block-rate-delay=950ms
+        - --hotstuff-timeout=2.15s
+        - --hotstuff-min-timeout=2.15s
+        - --ingress-addr=0.0.0.0:9000
+        - --insecure-access-api=false
+        - --access-node-ids=*
+      env:
+        - name: OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
+          value: http://tempo:4317
```

Comment:
You probably want to point this to a real tempo instance for benchnet.

Reply:
As of right now, this will require additional setup/configuration within the cluster. Once I have finished the cluster configuration, we will have a "real" endpoint that we can use. For the time being though, I think we can leave this as-is...we will just need to replace it with the real value later.